libs/@blockprotocol/graph/src/types/ontology/property-type.ts

```diff
@@ -12,3 +19,29 @@ export type PropertyTypeWithMetadata = {
   schema: PropertyType;
   metadata: OntologyElementMetadata;
 };
+
+export type AggregatePropertyTypesData = {
+  graphResolveDepths?: Partial<
+    Pick<
+      Subgraph<true>["depths"],
+      "constrainsValuesOn" | "constrainsPropertiesOn"
+    >
+  >;
+};
```

Comment:
Just to flag, this now makes this method inconsistent with `AggregateEntityTypes`. Also I've noticed you've made it `Partial`. 

Previously we had opted to make the whole depths object required, because we encountered too many scenarios where a dev absent-mindedly didn't think about what they actually needed. I appreciate that introduces a very large amount of boilerplate, so I could be swayed to us having them as optional. 

I think I actually was swayed in some previous discussion which is why `graphResolveDepths` is now optional in `getEntity` and `aggregateEntities`. However, it is not `Partial` (which _seems_ better to me if we're allowing it to be optional and presumably going to **specify** that it is 0 if you leave it undefined). So, all of that was a roundabout way to say:

**If we keep this can you go and change the other methods to be consistent with this approach?**

Reply:
Yes - https://github.com/blockprotocol/blockprotocol/pull/946/commits/086f506ad7e3054eed4c6896363cce4d53a06145